### DISCUSSION TITLE
genics-log-2

### DISCUSSION THREAD

**Discussion by giscus** [genicsblog/comments#128] [Announcements]
# genics-log-2

Here's the issue #2 in the Genics log series containing all the latest news and updates on our blog's changes from the recent months!

https://genicsblog.com/genics-log-2

**Comment by 0xMukesh** [comment]
noiceeeee

**Comment by TheBlapse** [comment]
Hello!

**Comment by shockz09** [comment]
nice blog

**Comment by anuraglol** [comment]
howdy

**Comment by AnishDe12020** [comment]
nice ser

**Comment by ankushchk** [comment]
cool
